# WorldObjects.msg: Observed objects in the world.

# Fields
# ======================================================================
WorldObject[] world_objects
